phpstan.neon.dist

```diff
@@ -8,6 +8,7 @@ parameters:
 	checkMissingCallableSignature: true
 
 	type_coverage:
-		return_type: 98
-		param_type: 98
-		property_type: 100
\ No newline at end of file
+		return_type: 100
+		param_type: 100
+		property_type: 100
+		print_suggestions: true
```

Comment:
ℹ️ This was missing so PHPStan would show exactly where the type hints are missing from